ansible/roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -16,6 +16,7 @@
     owner: root
     group: grafana
     mode: 0640
+    force: "{{ grafana.overwrite_config }}"
```

Comment:
is this going to be coerced correctly to a boolean if surrounded by double quotes? or will it be the case that `"false"` will be evaluated to `True` because the string is not empty?

Reply:
This is fine (necessary) for variables. If the value was unquoted, yaml would look at the first character after `:`, i.e. `{` and think that a dictionary follows.